dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -441,7 +441,7 @@
     }
 
     protected MethodConfig getMethodByName(String name) {
-        if (methods != null && methods.size() > 0) {
+        if (methods != null && !methods.isEmpty()) {
```

Comment:
**Scalability Issue**: Using '!methods.isEmpty()' is more readable and idiomatic than checking 'methods.size() > 0'. <br> **Fix**: Replace the condition 'methods.size() > 0' with '!methods.isEmpty()' for better readability. <br> **Code Suggestion**: 
 ```
 -        if (methods != null && methods.size() > 0) {
 +        if (methods != null && !methods.isEmpty()) {
 ```